README.md

```diff
@@ -229,3 +229,27 @@ The topic configs can be edited by supplying key value pairs as follows:
 ```bash
 kafkactl alter topic my-topic --config retention.ms=3600 --config cleanup.policy=compact
 ```
+
+### Consumer groups
+
+In order to get a list of consumer groups the `get consumer-groups` command can be used:
+```bash
+# all available consumer groups
+kafkactl get consumer-groups 
+# only consumer groups for a single topic
+kafkactl get consumer-groups my-topic
+# using command alias
+kafkactl get cg
+```
+
+To get detailed information about the consumer group use `describe consumer-group`. If the parameter `--partitions`
```

Comment:
`--partitions` seems to be not available

Reply:
I assume `--topic` is the inverse substitute? In that case it's missing in the README